helpers/remoteoauth/token.go

```diff
@@ -0,0 +1,166 @@
+package remoteoauth
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"sync"
+	"time"
+
+	cloudquery_api "github.com/cloudquery/cloudquery-api-go"
+	"github.com/google/uuid"
+	"golang.org/x/oauth2"
+)
+
+type Token struct {
+	accessToken string
```

Comment:
This 3 properties are logically grouped as an `oauth.Token`. Maybe `Token` should have an `oauth.Token` inside?

Reply:
Done in https://github.com/cloudquery/plugin-sdk/pull/1866/commits/75d68c7e2b44f1179d28514c6106bfd082415d3b